src/libos/src/time/up_time.rs

```diff
@@ -1,11 +1,12 @@
-use super::{do_clock_gettime, ClockID};
+use super::do_clock_gettime;
```

Comment:
Don't need to change.
`use super::{do_clock_gettime, ClockId};`

Reply:
Done